src/app/zap-templates/zcl/data-model/chip/service-area-cluster.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023-24 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <bitmap name="Feature" type="bitmap32">
+    <cluster code="0x0150" />
+    <field name="ListOrder"           mask="0x01" />
+    <field name="SelectWhileRunning"  mask="0x02" />
+  </bitmap>
+
+  <!-- Enum definitions for structures-->
+  <enum name="OperationalStatusEnum" type="enum8">
+    <cluster code="0x0150"/>
+    <item value="0x00" name="Pending"/>
+    <item value="0x01" name="Operating"/>
+    <item value="0x02" name="Skipped"/>
+    <item value="0x03" name="Completed"/>
+  </enum>
+
+  <!-- Data types -->
+  <struct name="LocationInfoStruct">
+    <cluster code="0x0150"/>
+    <item name="HomeLocationInfo"  type="HomeLocationStruct"    optional="false" isNullable="true"/>
+    <item name="LandmarkTag"       type="LandmarkTag"           optional="false" isNullable="true"/>
+    <item name="PositionTag"       type="PositionTag"           optional="false" isNullable="true"/>
+    <item name="SurfaceTag"        type="FloorSurfaceTag"       optional="false" isNullable="true"/>
+  </struct>
+
+  <struct name="MapStruct">
+    <cluster code="0x0150"/>
+    <item name="MapId"   type="int8u"                   optional="false"/>
```

Comment:
This does not match the spec.

Reply:
I'm not sure what does not match here. [This](https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/app_clusters/ServiceArea.adoc#52-mapstruct-type) is the definition I'm looking at.

Reply:
I have addressed this and all the other similar comments.